internal/service/team/data_source_team_test.go

```diff
@@ -91,3 +123,19 @@ func dataSourceConfigBasicByName(orgID, name, username string) string {
 		}
 	`, orgID, name, username)
 }
+
+func dataSourceConfigNoUsers(orgID, name string) string {
+	return fmt.Sprintf(`
+		resource "mongodbatlas_team" "test" {
+			org_id     = "%s"
+			name       = "%s"
```

Comment:
```suggestion
			org_id     = %[1]q
			name       = %[2]q
```

q for quoted strings, use order to make it easier to read and less error-prone.

Reply:
Thanks for the suggestion. Changed it also in the other configs in that same file [9b067bc](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/3483/commits/9b067bc91810ae92865d22dd82d08c6a7a505d40)